rules/windows/process_creation/proc_creation_win_cmdkey_recon.yml

```diff
@@ -21,7 +21,13 @@ detection:
     selection_cli:
         CommandLine|contains:
             - ' /list'
+            - ' /lis'
+            - ' /li'
+            - ' /l'
             - ' -list'
+            - ' -lis'
+            - ' -li'
+            - ' -l'
     condition: all of selection*
```

Comment:
As it is contains  `' -l'`  is  ' -l*'  so it covert all li, lis, list
Same for `' /l'`